.github/workflows/ci.yml

```diff
@@ -249,6 +256,7 @@ jobs:
       - test-linux
       - test-windows
       - check
+      - package
```

Comment:
This is ok... but maybe `pypi-publish` should depende on `package` .

Maybe we want to block a release if the package is not valid.